app/src/components/Account.tsx

```diff
@@ -1,15 +1,58 @@
 import React from 'react';
 
+import { truncateMiddle } from '@kibalabs/core';
 import { Alignment, Box, Button, Direction, Image, Stack, Text } from '@kibalabs/ui-react';
+import { ethers } from 'ethers';
+import { SiweMessage } from 'siwe';
+
+const domain = window.location.host;
+const origin = window.location.origin;
+const provider = new ethers.providers.Web3Provider(window.ethereum);
+const signer = provider.getSigner();
+
+const createSiweMessage = (address: string, statement: string) => {
+  const message = new SiweMessage({
+    domain,
+    address,
+    statement,
+    uri: origin,
+    version: '1',
+    chainId: 1,
+  });
+  return message.prepareMessage();
+};
 
 export const Account = (): React.ReactElement => {
+  const [account, setAccount] = React.useState<string | undefined>(undefined);
```

Comment:
this stuff should all be in navbar (or ideally now app.tsx). the Account component should only show the image and account name. We will have multiple connect buttons throughout the app and wont repeast this functionality everywhere we will have it in just one place (ideally app.tsx)